internal/events/event.go

```diff
@@ -21,7 +22,7 @@ type Handler interface {
 	//     host added to cluster, we have the host-id as the main entityID and
 	//     the cluster-id as another ID that this event should be related to
 	// otherEntities arguments provides for specifying mor IDs that are relevant for this event
-	AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time)
+	AddEvent(ctx context.Context, clusterID strfmt.UUID, hostID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{})
```

Comment:
can you please add a comment with prop examples and also some explanation about this field? something similar to what you have in your testing

Reply:
I will when I'll added the actual implementation, so to make sure it is accurate